### PR TITLE
Style colors

### PR DESCRIPTION
**Pull request by rebecalsantos**
-home /booking screen/deleting booking
-WeeklyTable
- buttons

### CODE REVIEW

File: client/src/pages/WeeklyTable.css [hunk 1]
```diff
@@ -28,8 +28,8 @@ th {
 	margin-right: 0;
 	width: 100%;
 	margin-top: 20px;
-	background-color: #ddd4d2;
-	color: black;
+	background-color: #f8f3f2;
```
Comment (gregdyke) [inline client/src/pages/WeeklyTable.css]: I think this looks a bit weird in the end :/ so I wouldn't include these changes

**Review by gregdyke — CHANGES_REQUESTED** [review]
I hope we can get these changes in before the demo :)

File: client/src/pages/WeeklyTable.css (hunk 1 above)
Comment (gregdyke) [reply client/src/pages/WeeklyTable.css]: I mean this color and background color in the weeklytable

Comment (Benjamin-GitHub) [reply client/src/pages/WeeklyTable.css]: I think both were great before, also more readable

File: client/src/pages/WeeklyTable.css [hunk 2]
```diff
@@ -15,21 +15,21 @@ td.desk-column {
...
 }
 
 .table-container #table {
 	margin-right: 0;
 	width: 100%;
 	margin-top: 20px;
-	background-color: #f8f3f2;
-	color: #737872;
+	background-color: #ddd4d2;
+	color: black;
```
Comment (Benjamin-GitHub) [inline client/src/pages/WeeklyTable.css:32]: thanks. better!

File: client/src/pages/ModalBookingScreen.js [hunk 3]
```diff
@@ -98,8 +98,8 @@ function ModalBookingScreen({
 			open={open}
 			trigger={
 				<Icon.Group>
-					<Icon name="map marker alternate" inverted color="orange" />
-					<Icon corner name="add" color="orange" />
+					<Icon link name="map marker alternate" inverted color="orange" />
+					<Icon link corner name="add" color="orange" />
```
Comment (Benjamin-GitHub) [inline client/src/pages/ModalBookingScreen.js:102]: I didn't get at the all search this link what is exactly for!! with or without functionality is the same!?

**Review by Benjamin-GitHub — APPROVED** [review]

**Review by gregdyke — APPROVED** [review]

**Pull request merged**